build-system/compile/compile.js

```diff
@@ -328,8 +322,13 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
         'constantProperty',
         'globalThis'
       );
+      compilerOptions.jscomp_warning.push('accessControls');
+      compilerOptions.jscomp_off.push('moduleLoad', 'unknownDefines');
       compilerOptions.conformance_configs =
         'build-system/conformance-config.textproto';
+    } else {
+      compilerOptions.jscomp_warning.push('accessControls', 'moduleLoad');
```

Comment:
they should be on by default or no?

Reply:
Defaults are different for each type. This makes both of them warnings, which is what we want.